2 py 1.py

```diff
@@ -0,0 +1,7 @@
+# 1 – Напишите программу, которая принимает на вход вещественное число и 
+# показывает сумму его цифр.
+# Пример:
+# 6782 -> 23
+# 0,56 -> 11
+
+print(sum(map(int, (x for x in input('Введите вещественное число: ') if x.isdigit()))))
```

Comment:
```suggestion
print(sum(int(x) for x in input('Введите вещественное число: ') if x.isdigit()))
```
Так ведь чуть лучше, правда?